packages/web-helpers/src/WebUser.ts

```diff
@@ -234,69 +289,39 @@ export class WebUser extends EventEmitter {
         const messageType = "inbox/type/message"
         const config: MessageSendConfig = {
             did,
-            recipientContextName: "Verida: Vault"
-        }
-
-        // Send the message across the network
-        await messaging.send(did, messageType, data, message.subject, config)
-    }
-
-    /**
-     * Is a user connected?
-     * 
-     * Will auto-connect the user from local storage session if it exists.
-     * 
-     * @returns 
-     */
-    public async isConnected(): Promise<boolean> {
-        if (this.did) {
-            return true
-        }
-
-        if (hasSession(this.config.contextConfig.name)) {
-            const connected = await this.connect()
-            return connected
+            recipientContextName: VAULT_CONTEXT_NAME
         }
 
-        return false
+        // Send the message across the Network
+        return messaging.send(did, messageType, data, message.subject, config)
     }
 
     /**
-     * Throw an exception if a user isn't connected
-     */
-    private async requireConnection(): Promise<void> {
-        const isConnected = await this.isConnected()
-        if (!isConnected) {
-            throw new Error('Not connected to Verida network')
-        }
-    }
-
-    /**
-     * Open a datastore owned by this user
-     * 
-     * @param schemaURL 
-     * @param config 
-     * @returns 
+     * Open a datastore owned by this user.
+     *
+     * @param {string} schemaURL
+     * @param {DatastoreOpenConfig} config
+     * @returns A Promise that will resolve to the datastore instance.
      */
-    public async openDatastore(schemaURL: string, config?: DatastoreOpenConfig): Promise<IDatastore> {
-        const context = await this.getContext()
-        return await context.openDatastore(schemaURL, config)
+    public openDatastore(schemaURL: string, config?: DatastoreOpenConfig) {
```

Comment:
Must be async

Reply:
It is async, we just don't need the syntax as it's inferred by the return of the Promise from `context.openDatastore`.

Reply:
I'm confused by this, see Discord thread. We can summarise that conversation here once concluded.